packages/data/walkthroughs/9.9.9.json

```diff
@@ -1583,6 +1805,15 @@
     "R11": {
       "resultDisplayMessage": "Based on the provided building characteristics the dwelling unit must be provided with:<ul><li>an exterior passageway that has a floor assembly with a <defined-term>fire-resistance rating</defined-term> less than 45 min, and</li><li>be served by a single <defined-term>exit</defined-term> stairway or <defined-term>ramp</defined-term>, and</li><li>be located more than 1.5 m above adjacent ground level, to meet the provisions of the BC Building Code 2024.</li></ul>"
     },
+    "R1.0-remove-after-review-9.9.9.3": {
+      "resultDisplayMessage": "The dwelling unit meets the requirement of sufficient means of egress as per BC Building Code 2024."
+    },
+    "R1.1-remove-after-review-9.9.9.3": {
+      "resultDisplayMessage": "The dwelling unit is located above another dwelling unit or common space. Specific requirements apply."
+    },
+    "R2.0-remove-after-review-9.9.9.3": {
+      "resultDisplayMessage": "The dwelling unit does not meet the requirement of sufficient means of egress as per BC Building Code 2024."
+    },
```

Comment:
I believe these can be removed. I think that they are redundant of R10, R11, and R20.